README.md

```diff
@@ -180,6 +300,84 @@ Localization of the editor is an ongoing effort, the main target usage of embedd
+The other way of triggering the plugin is to write one of the trigger phrases described in the plugin docs:
+* [specification]**decreet** [words to search for] *(e.g. "gemeentedecreet wijziging")*
+* **omzendbrief** [words to search for]
+* **verdrag** [words to search for]
+* **grondwetswijziging** [words to search for]
+* **samenwerkingsakkoord** [words to search for]
+* [specification]**wetboek** [words to search for]
+* **protocol** [words to search for]
+* **besluit van de vlaamse regering** [words to search for]
+* **gecoordineerde wetten** [words to search for]
+* [specification]**wet** [words to search for] *(e.g. "kieswet wijziging", or "grondwet")*
+* **koninklijk besluit** [words to search for]
+* **ministerieel besluit** [words to search for]
+* **genummerd besluit** [words to search for]
+
+After writting this a little card will appear on the right that you can expand to the big modal.
+
+![citation plugin](https://imgur.com/oerd9rV.png)
+
+### Roadsign Regulation
+For this plugin you will need to be in a besluit with one of these types:
+* `https://data.vlaanderen.be/id/concept/BesluitType/4d8f678a-6fa4-4d5f-a2a1-80974e43bf34`
+* `https://data.vlaanderen.be/id/concept/BesluitType/7d95fd2e-3cc9-4a4c-a58e-0fbc408c2f9b`
+* `https://data.vlaanderen.be/id/concept/BesluitType/3bba9f10-faff-49a6-acaa-85af7f2199a3`
+* `https://data.vlaanderen.be/id/concept/BesluitType/0d1278af-b69e-4152-a418-ec5cfd1c7d0b`
+* `https://data.vlaanderen.be/id/concept/BesluitType/e8afe7c5-9640-4db8-8f74-3f023bec3241`
+* `https://data.vlaanderen.be/id/concept/BesluitType/256bd04a-b74b-4f2a-8f5d-14dda4765af9`
+* `https://data.vlaanderen.be/id/concept/BesluitType/67378dd0-5413-474b-8996-d992ef81637a`
+
+If this condition is met you will find a button called `Voeg mobiliteitsmaatregel in` in the insert menu. When this button is clicked a modal will apear where you can filter and select which roadsign regulation you want to insert.
+
+![roadsign regulation modal](https://i.imgur.com/z7My8lm.png)
+
+### Variable
+This plugin allows you to insert variables in the document, a variable is a value to be filled by another user, ideally you will have 2 separate instances of the editor, the first one will enable this plugin and the other will enable the template-variable plugin.
+In order to insert a variable you will need to use the insert variable card.
+
+![insert variable card](https://imgur.com/9kSqgXc.png)
+
+In this dropdown you can select one of the 4 default types of variables (or more if you added your custom variables to the config): text, number, date or codelist. If codelist is selected you will need to specify which codelist you want to use
+
+### Template variable
+This plugin allows to fill the codelist variables generated by the variable plugin, if you are in one of these variables the following card will appear:
+
+![template variable card](https://imgur.com/b5Kmmqj.png)
+
+Using this card you will be able to select one of the codelist values.
```

Comment:
```suggestion
Using this card you will be able to select one of the codelist values. If in a date variable, a date-picker will be shown instead.

Other value types (text and number) can simply be filled in in the main editor. As of yet there is no validation on the number type.
```

Reply:
See previous comment, the dates are not filled with this plugin. I agree with the last line